swagger.yaml

```diff
@@ -1937,20 +1937,6 @@ paths:
                       "start": "2020-12-22T14:15:00Z",
                       "end": "2021-03-22T14:15:00Z"
                     },
-                    "deeplink": [
```

Comment:
We could put the "portal" URL in here, as we have that, e.g. https://www.bbc.co.uk/iplayer/episode/b04vfc5g

Reply:
I've generically called it links